### PR TITLE
#21: Refactored integration tests using test-db-builder-java

### LINKED ISSUE #21: Use test-db-builder-java to create database objects in test
Add [test-db-builder-java](https://github.com/exasol/test-db-builder-java) to create the tables and udf scripts in tests.

### PR DESCRIPTION
**Pull request by morazow**
Fixes #21.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ env:
 
 before_install:
   - git fetch --tags
-  - docker pull exasol/docker-db:7.0.4
+  - docker pull exasol/docker-db:7.0.5
```
Comment (AnastasiiaSergienko) [inline .travis.yml:18]: Should we use the latest 7.0.6?

File: doc/changes/changes_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* #21: Refactored integration tests using test-db-builder-java
+
+## Dependency updates
+
+* Updated `com.amazonaws:aws-java-sdk-core:1.11.916` to `1.11.942`
+* Updated `com.amazonaws:aws-java-sdk-kinesis:1.11.916` to `1.11.942`
+* Updated `com.exasol:exasol-testcontainers:3.4.0` to `3.4.1`
+* Updated `org.testcontainers:localstack:1.15.0` to `1.15.1`
+* Updated `org.mockito:mockito-core:3.5.10` to `3.7.7`
+* Updated `org.mockito:mockito-core:3.5.10` to `3.7.7`
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.0.md:16]: Duplicate of the previos line

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala [hunk 3]
```diff
@@ -17,13 +20,16 @@ trait KinesisAbstractIntegrationTest extends AnyFunSuite with BeforeAndAfterAll
   val JAR_NAME_PATTERN = "exasol-kinesis-connector-extension-"
   val DOCKER_IP_ADDRESS = "172.17.0.1"
   val TEST_SCHEMA_NAME = "kinesis_schema"
-  var assembledJarName: String = _
 
-  val exasolContainer = new ExasolContainer("7.0.4")
+  val exasolContainer = new ExasolContainer("7.0.5")
```
Comment (AnastasiiaSergienko) [inline src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala:24]: If we update Exasol, don't forget to update here too

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]
Looks good, just a couple of small comments

File: .travis.yml (hunk 1 above)
Comment (morazow) [reply .travis.yml:18]: Good idea!
Forgot that we had another release.

File: doc/changes/changes_1.0.0.md (hunk 2 above)
Comment (morazow) [reply doc/changes/changes_1.0.0.md:16]: Fixed.

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**